aspnetcore/host-and-deploy/iis/index.md

```diff
@@ -518,7 +518,7 @@ If the IIS worker process requires elevated access to the app, modify the Access
 
 1. Select the **Locations** button and make sure the system is selected.
 
-1. Enter **IIS AppPool\\<app_pool_name>** in **Enter the object names to select** area. Select the **Check Names** button. For the *DefaultAppPool* check the names using **IIS AppPool\DefaultAppPool**. When the **Check Names** button is selected, a value of **DefaultAppPool** is indicated in the object names area. It isn't possible to enter the app pool name directly into the object names area. Use the **IIS AppPool\\<app_pool_name>** format when checking for the object name.
+1. Enter **IIS AppPool\\\<app_pool_name>** in **Enter the object names to select** area. Select the **Check Names** button. For the *DefaultAppPool* check the names using **IIS AppPool\DefaultAppPool**. When the **Check Names** button is selected, a value of **DefaultAppPool** is indicated in the object names area. It isn't possible to enter the app pool name directly into the object names area. Use the **IIS AppPool\\\<app_pool_name>** format when checking for the object name.
```

Comment:
Why are three backslashes needed here? The original with two seems to render correctly as one.

Reply:
We were (last I heard ... it might be a 🧀-moved scenario for me) that we were escaping the < char, too. Therefore, the first `\\` yields a `\` and the `\<` yields a `<`. Net result ... `\<`.

Reply:
... and I agree on what's going on with the rendering. It looks like that first `\` is really coming out as `\`. Therefore, I assume that `\` is fine to get a `\` based on the rendered behavior ... but didn't we have problems with that at one point? I thought that unless it went in as `\\` it didn't work. Perhaps, there's an odd scenario here where if the `\\<` in just this one case works out but other `\` don't ... maybe some kind of parsing algorithm thing allows it to work.

Reply:
How does it look based on this PR?

https://review.docs.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/index?view=aspnetcore-2.1&branch=pr-en-us-12647#application-pool-identity

Reply:
It renders correctly in both production and staging, which doesn't make sense to me, so I guess we could go either way ... Maybe the rendering engine is getting smarter about not swallowing unescaped  `<`  in which case some or all of the other changes in this PR may not be needed.

Reply:
Yes, I'm mostly just going by convention these days. I think Scott knows more about it. We talked about this ages ago. Check out the Example 224 here ...

https://spec.commonmark.org/0.15/#backslash-escapes

The only one that I wasn't doing was the `#` when it appears mid-sentence. I think it would only be a problem if it starts a line and we wanted it to show up ... and that never has happened AFAIK.

... and check out the next one ... Example 225 ... it seems to indicate to me if I'm reading that correctly that the correct way to go **_IS_** `\\\<`.

Reply:
Not exactly - that just confirms that triple backslash will work, not that we need to do it.  Escaping is an option that is available to make things render the way we intend them to.  IF they render correctly without the escaping, we're just adding noise to the text.  I'd like to understand why the escaping isn't needed in these instances, though.  Does the rendering engine now swallow only valid HTML elements and everything else no longer needs escaping?

Reply:
> not that we need to do it

I think that we should escape all backslashes. In other cases (earlier ... months and months ago when discussed prior) I think we had to escape backslashes in cases where they had a non-alphanumeric char behind them.

I agree that the `\\<` working properly is *very strange* ...... 👽? lol I actually would expect that to render as a backslash followed by **_a tag_** that doesn't show up.

Another side note here is that discussions led to using the brace `{` with text instead instead of angle brackets and underscores. The thinking there is that we get our paths localized. So ... it was like ...

* `/\<A_PLACEHOLDER_WITH_UNDERSCORES>` :x: *Bad*
* `/{A PLACEHOLDER WITHOUT UNDERSCORES}` ✔️ *Good*

... when not code-fenced, it should (in theory 🤞) localize well.

Reply:
If the markdownlint rule is reporting false positives, another option is to repurpose this PR to make it disable the rule instead.  I'm not sure the benefit of this rule outweighs the nuisance factor, as any time we use inline HTML we'll find out quickly enough if the rendering engine has a problem with it.

Reply:
I'm in agreement with Tom.

Reply:
@scottaddie ... but WRT the backslash: Are there conventions/rules/scenarios to escape and not to escape, or was it decided back then that we're always going to escape it?

Reply:
Looks like there is a new release of Markdownlint that changed the regex https://github.com/DavidAnson/markdownlint/issues/188
I'll wait till the CLI library is updated and re-run this